.github/workflows/aws-cleanup.yaml

```diff
@@ -0,0 +1,31 @@
+name: AWS cleanup
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
Why only once a week?

Reply:
I figured we don't want to cleanup everyday as sometimes the leftovers from a failed run can be used for debugging